bangazon-clumsy-penguins/BangazonSite
# Ajw default value

# Name - Default Product Category 

## Description
When creating a new product to sell the default category will be "None"

User cannot submit if the product type is set to none

## Steps to Test
When user is logged in

Click on Sell

Verify that the initial product type is "None" in the drop down

Will out all form fields but set Product Type to none. Verify that the form doesn't submit

##### Link to Feature Ticket
#5
